tests/integration/__snapshots__/formatExports.test.ts.snap

```diff
@@ -2471,8 +2471,8 @@ exports[`SPDX JSON alpine 1`] = `
    "name": "libvncserver",
    "SPDXID": "redacted",
    "versionInfo": "0.9.9",
-   "supplier": "Person: A. Wilcox \\u003cawilfox@adelielinux.org\\u003e",
-   "originator": "Person: A. Wilcox \\u003cawilfox@adelielinux.org\\u003e",
+   "supplier": "Person: A. Wilcox <awilfox@adelielinux.org>",
```

Comment:
It looks like we fixed a couple encoding bugs in Syft SPDX JSON outputs? Would like a second set of eyes, since I wasn't expected a snapshot diff here.